### ISSUE TITLE
Site header has restricted height

### ISSUE THREAD

**Issue by joedolson** [wpaccessibility/wp-a11y-docs#44]
The `header` container has a fixed height of 3.75rem, which causes the text to overflow when text spacing is adjusted, violating 1.4.12: Text Spacing.

This is apparently done to keep the header height aligned with the search bar and the top of the nav aligned with the top of the text body; but I think that's unimportant.

Fix `.site-header { height: 3.75rem; max-height: 3.75rem; }`
